web/search.dart

```diff
@@ -139,16 +140,19 @@ List<IndexItem> findMatches(List<IndexItem> index, String query) {
 }
 
 const minLength = 1;
-const suggestionLimit = 10;
+var suggestionLimit = 10;
+var allResults = 0;
```

Comment:
The name here makes it sound like this is a list or something. I think we prefer a noun like "length" or "count" in the name for an int like this.